tool/tsh/tsh.go

```diff
@@ -423,7 +427,7 @@ func Run(args []string, opts ...cliOption) error {
 	app.HelpFlag.Short('h')
 
 	ver := app.Command("version", "Print the version")
-
+	ver.Flag("format", "Format output (text, json, yaml)").Short('f').Default(teleport.Text).EnumVar(&cf.Format, defaultFormats...)
```

Comment:
Do we support short version of this flag `-f` everywhere? I don't recall. If there are some commands that don't support short flag (e.g. due to a conflict with another flag's short name), I'd probably just keep the long `--format` version, for consistency.

Reply:
Also, use `strings.Join(defaultFormats, ", ")` instead of verbatim `(text, json, yaml)` list.